ios-base/Networking/Services/UserService.swift

```diff
@@ -96,15 +104,32 @@ class UserService: BaseApiService<UserResource> {
   }
 
   func logout(
-    _ success: @escaping () -> Void, failure: @escaping (_ error: Error)
+    _ success: @escaping () -> Void,
+    failure: @escaping (_ error: Error)
   -> Void) {
-    request(for: UserResource.logout,
-            onSuccess: { _ in
-              UserDataManager.deleteUser()
-              SessionManager.deleteSession()
-              success()
-            }, onFailure: { error, _ in
-              failure(error)
-            })
+    request(
+      for: UserResource.logout,
+      onSuccess: { _ in
+        UserDataManager.deleteUser()
+        SessionManager.deleteSession()
+        success()
+    }, onFailure: { error, _ in
+      failure(error)
+    })
+  }
+  
+  func deleteAccount(
+    _ success: @escaping () -> Void,
+    failure: @escaping (_ error: Error)
+  -> Void) {
```

Comment:
As in the logout function, the line breaking here seems odd. There are some inconsistencies across the file that we should address.